### ISSUE TITLE
No Audio

### ISSUE THREAD

**Issue by RjBass3** [Leviathan5/BeamSoundlyInteractive#27]
No matter what I look at, I can't get Soundwave to play any audio.  Not locally or on stream.

**Comment by DeekyJay** [comment]
@RjBass3 In the settings tab, please check your audio output device selection. Test different options and see if others work.

**Comment by RjBass3** [comment]
Ok Well it half way works now.  I closed the app and re-opened it.  Some of my sounds I can hear, others I can't but people watching the stream can hear them all.  One sound effect  charges people a spark but doesnt play.  The main fix for the audio issue originally though was that my sound effects were coming from a network drive.  When I moved them to my PC they started working.  

Checking the audio output was the first thing I looked at.
